bin/runtime/src/chain_extension/snarcos-extension/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "snarcos-extension"
```

Comment:
I find it kinda weird this crate is under `runtime` folder. `contracts` would be much better place in my opinion

Reply:
I have to plead guilty here as I made @pmikolajczyk41 move it from `/contracts` to here. 

My reasoning was (and still is) that the chain extension is part of the core chain (you can see it being defined in [`runtime/src/lib.rs` ](https://github.com/Cardinal-Cryptography/aleph-node/blob/8d56f701304fd43e1a018bcf9c5671d9ef55f5ab/bin/runtime/src/lib.rs#L680) ) and not living in the client space like contracts. Now it lives closer to the place where it's actually used and defined - we wouldn't be adding a chain extension written by clients (`/contracts`, even though now live in the main repo are not part of the core chain). 

We can still make it "publishable" and have contracts refer to it in their dependencies https://github.com/Cardinal-Cryptography/aleph-node/blob/liminal/chain-extension/contracts/snarcos-contract/Cargo.toml#L26 - making it all type-safe.

Reply:
I see both points and I really cannot decide. There is, lets say, some solution with using https://github.com/Supercolony-net/obce, but for now it would be an overkill. 

I'll leave this as it is now, and in case we find it inconvenient, we'll move it somewhere else (I suppose/hope somebody will play with it more and thus have some experience-based feedback).

CCing @Yithis to make him aware of these doubts (since he'll be extending this extension (today?))